### PR TITLE
Support psr/log 2, 3

### PR DESCRIPTION
**Pull request by recca0120**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

#### What's in this PR?

support psr/log 2, 3

#### Why?

some third packages require psr/log 3

### CODE REVIEW

**Review by dbu — APPROVED** [review]
awesome, thanks!

can you please fix the codestyle things in the stub?

the code coverage error is a setup problem and not related to your pull request, we can ignore that one here. (and i think my solution will be to drop that test, i don't think anyone looks at the coverage metrics of this package)

**Comment by recca0120** [comment]
fixed

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dbu) [inline .idea/workspace.xml]: this should be in your global gitignore configuration ;-)

can you please remove it from the pull request?

**Comment by dbu** [comment]
apart from the intellij configuration file, this looks good now :+1: 

i will merge as soon as you can remove that.

File: .idea/workspace.xml (hunk 1 above)
Comment (recca0120) [reply .idea/workspace.xml]: cool. I'll add this to my global gitignore configuration.
and .idea remoed already

**Comment by dbu** [comment]
thank you! 

tagged as https://github.com/php-http/vcr-plugin/releases/tag/1.2.2

**Pull request merged**